apps/base-docs/docs/pages/builderkits/onchainkit/appchain/bridge.mdx

```diff
@@ -0,0 +1,388 @@
+---
+title: <AppchainBridge /> · OnchainKit
+description: Bridge to appchains with OnchainKit
+---
+
+import { DefaultAppchainBridgeComponent } from '@/components/AppchainComponents';
+
+# `<AppchainBridge />`
+
+:::danger[⚠️ NOTE]
+The `AppchainBridge` component is alpha release software and is provided AS-IS. Use at your own risk.
+:::
+
+The `AppchainBridge` component provides a simple interface for implementing bridging to appchains with OnchainKit.
+
+## Prerequisites
+
+Before using the `AppchainBridge` component, ensure you've completed the [Getting Started](/builderkits/onchainkit/getting-started) steps.
+
+### Starting a new project
+
+If you're starting a new project, we recommend using `create onchain` to scaffold your project.
+
+```bash
+npm create onchain@latest
+```
+
+### Adding to an existing project
+
+If you're adding `AppchainBridge` to an existing project, simply install OnchainKit.
+
+:::code-group
+
+```bash [npm]
+npm install @coinbase/onchainkit
+```
+
+```bash [yarn]
+yarn add @coinbase/onchainkit
+```
+
+```bash [pnpm]
+pnpm add @coinbase/onchainkit
+```
+
+```bash [bun]
+bun add @coinbase/onchainkit
+```
+
+:::
+
+Wrap the `<OnchainKitProvider />` around your app, following the steps in [Getting Started](/builderkits/onchainkit/getting-started#add-providers).
```

Comment:
add-providers is now nested within the manual installations, so perhaps `builderkits/onchainkit/installation/nextjs#add-providers` is the best link? 

or `builderkits/onchainkit/getting-started#manual-installation`